platform/linux-generic/odp_traffic_mngr.c

```diff
@@ -2542,8 +2542,8 @@ void odp_tm_egress_init(odp_tm_egress_t *egress)
 	memset(egress, 0, sizeof(odp_tm_egress_t));
 }
 
-int odp_tm_capabilities(odp_tm_capabilities_t capabilities[] ODP_UNUSED,
-			uint32_t              capabilities_size)
+static int tm_capabilities(odp_tm_capabilities_t capabilities[],
```

Comment:
Changing this function to an internal one and creating a new odp_tm_capabilities function that just calls this seems totally unnecessary to me. I do not see why this could not be left as it is (apart from removing the incorrect ODP_UNUSED attribute).

Reply:
You mean, I should use odp_tm_capabilities() as it is in odp_tm_egress_capabilities() ?

Reply:
Yes, why not?

Reply:
Ack, just wanted to confirm.

Reply:
I realized later that in the last patch, we are deprecating odp_tm_capabilities(). So using it in odp_tm_egress_capabilities() as ODP_DEPRECATED(odp_tm_capabilities) then is right I guess ?

Reply:
I was thinking about the patch that introduced the change and forgot the deprecation in the later patch. So yes, the new internal function is needed. Having the change now in the last patch is logical.

Reply:
Ack.